### PR TITLE
Added support for bus statistics in the kvaser interface.

### PR DESCRIPTION
**Pull request by sIuv**

### CODE REVIEW

File: can/interfaces/kvaser/structures.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+
+"""
+Contains Python equivalents of the structures in CANLIB's canstat.h, 
+with some supporting functionality specific to Python.
+
+Copyright (C) 2010 Dynamic Controls
```
Comment (hardbyte) [inline can/interfaces/kvaser/structures.py]: Unless I'm missing something this isn't required

Comment (sIuv) [reply can/interfaces/kvaser/structures.py]: Thank you for the fast response. Noticed that the reference to canstat.h was not correct either.

**Comment by sIuv** [comment]
For documentation, would it be appropriate to remove get_stats form the bus section and add a new section.
E.g. named "Custom methods" and include it there instead.

Bus
~~~~~~~~~~~~~~~~~~~

.. autoclass <...>.KvaserBus
	:members:
	:undoc-members: get_stats

Custom methods
~~~~~~~~~~~~~~~~~~

.. automethod <...>.KvaserBus.getstats

**Comment by hardbyte** [comment]
Yeah documenting custom methods like that sounds like a good approach.

**Comment by sIuv** [comment]
Moved get stats to a separate documentation section like the suggestion above.

**Comment by nitishn23** [comment]
Hello Can you  please suggest is this good way to get bus statistics for the Kvaser interface, i am using "**can.interface.Bus.state()**"this method  to get bus state to indicate of bus status unfortunately it always returns as Active when the actual state is Passive. I checked the "**can.interface.get_stats()**" it does return with high bus load when the connection is lost with some error frames.

**Pull request merged**